### PR TITLE
Up-to-date the full mobile-version

### PR DESCRIPTION
**Pull request by aradradev**
### The last add for to the responsive
- Added a full mobile responsive web-design
-  Added the UI design from figma
- Added a form to reach out me

### CODE REVIEW

**Review by Owusu-Desmond — CHANGES_REQUESTED** [review]
Hi @aradradev , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Required Changes ♻️
_Check the comments under the review._

- [ ] I can't use inline comments for this suggestion but in your index.html you use `h1`  for the heading "My Recent Works". why don't you consider using other heading tags such as `h2`, `h3` ... since `h1` have to be used only once in your HTML document :smile: 
- [ ] I can't use inline comments for this suggestion but in your index.html(The work section), make sure that the list of project technologies (e.g., "JavasScript", "HTML" and "CSS") is a list `ul` and each of them a `li`

- [ ]  In your index.html(the about me section), make sure that the technology categories ("Languages", "Frameworks" and "Skills") are in a `ul`, each of them in a `li` and he technology in each category(e.g., "JavasScript", "HTML" and "CSS") are also a list `ul` and each of them a `li`

- [ ] I can't use inline comments for this suggestion but in your index.html you use `h1`  for the heading "About me". why don't you consider using other heading tags such as `h2`, `h3` ... since `h1` have to be used only once in your HTML document :smile: 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -177,31 +191,37 @@ <h1 class="big-title big-title-about">About me</h1>
...
-                <p>JavaScript</p>
-                <p>Ruby</p>
-                <p>HTML</p>
-                <p>CSS</p>
+                <ul>
+                    <li>JavaScript</li>
+                    <li>Ruby</li>
+                    <li>HTML</li>
+                    <li>CSS</li>
+                </ul>
```
Comment (anagudelogu) [inline index.html:199]: - [ ] You did a great job working on the previous reviewer's suggestions! 🚀 However, I think that what the reviewer meant was that there's an "outer" list: (Languages, Frameworks, Skills) and inside each item of that list, there's another one listing its contents, something like this:

```html
<ul>
  <li>
    <h4>Languages</h4>
    <ul>
      <li>JavaScript</li>
      <li>Ruby</li>
      <li>HTML</li>
      <li>CSS</li>
    </ul>
  </li>
  <li>
    <h4>Frameworks</h4>
    ...
  </li>
  <li>
    <h4>Skills</h4>
    ...
  </li>
</ul>
```

Please consider applying this change! 🤓 🔧

File: style.css [hunk 2]
```diff
@@ -374,6 +374,7 @@ button {
 .form-input textarea {
   border: none;
   outline: none;
+  margin: 0.2rem;
   padding: 0.5rem;
   font-weight: 400;
```
[the remark this answers is not in the document]
Comment (anagudelogu) [reply style.css:400]: - [ ] Good job adding the button interaction effects, however, the enabled and hover states are not correct, please consider fixing it! 🤓 🔧 

[Screencast from 28-04-23 10:22:04.webm](https://user-images.githubusercontent.com/77069011/235190352-2c4f6b67-628d-40c8-bbe9-5be0fa91fd06.webm)

**Review by anagudelogu — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED ♻️ 🚧 

Hi @aradradev 👋🏼 
I'm Andres.

You've done a great job implementing some of the requested changes for this milestone 🚀, but there are still some which haven't been addressed yet. You're getting closer, keep up the good work! 🤖 🎆 



 ## Highlights
✅ Great use of GitHub flow. 💯 
✅ Your project is documented professionally. 📜 
✅ Great UI design. 💻 
✅ Descriptive PR title, summary and commit messages. 💬 


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Abenezer-Tilahun — APPROVED** [review]
Hi @aradradev 
 
Your project is complete  👍 💯   There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct GitHub -Flow.
✔️ Professional README  
✔️ Good Commit Messages
✔️ Good job for Implemented Suggested changes.


## Optional suggestions 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by aradradev** [comment]
Time to merge it finally.

**Pull request merged**